Dockerfile.dapper

```diff
@@ -1,6 +1,6 @@
-FROM golang:1.10
+FROM golang:1.12
 RUN apt-get update && \
-    apt-get install -y xz-utils zip rsync
+    apt-get install -y xz-utils zip rsync musl-dev
```

Comment:
We had faced some issues with `musl` in the past. Curious, what are you using this for?

Reply:
I found the build was not successful without it on golang 1.12. I’ve been running it in production for a month or so without issue but equally have no specific requirements for it with the code changes I’ve made.

Reply:
@leodotcloud I've done some more testing and it's not necessary - I've removed it